proposals/records.md

```diff
@@ -48,9 +48,9 @@ The synthesized members are as follows:
 
 If the record is derived from `object`, the record type includes a synthesized readonly property equivalent to a property declared as follows:
 ```C#
-protected Type EqualityContract { get; };
+Type EqualityContract { get; };
 ```
-The property is `virtual` unless the record type is `sealed`.
+The property is `virtual` and `protected` unless the record type is `sealed`. Otherwise, the property is `private`.
```

Comment:
Having unless and otherwise like this doesn't read particularly well. I'd suggest putting the exception first, like this:

```suggestion
The property is `private` if the record type is `sealed`. Otherwise, the property is `virtual` and `protected`.
```